typed-store/src/rocks/mod.rs

```diff
@@ -21,6 +21,37 @@ mod tests;
 type DBRawIteratorMultiThreaded<'a> =
     rocksdb::DBRawIteratorWithThreadMode<'a, DBWithThreadMode<MultiThreaded>>;
 
+#[macro_export]
+// It generates a method (reopencfs) to reopen a list of column families.
+// The macro should be called by passing in the column family names and the
+// corresponding DBMap Key-Value types.
+//
+// Example: "cf_1";<String, u32>, "cf_2";<u32, u32>
```

Comment:
It would be great to add a [documentation test](https://doc.rust-lang.org/rustdoc/documentation-tests.html) here.

Reply:
Absolutely! Wasn't aware that Rust offers "Documentation test" - that is awesome!